docs/quick-start.MD

```diff
@@ -1144,8 +1144,71 @@ spec:
   endpoints: []
 ```
 
+### prometheus Rule duplcation
```

Comment:
could you also describe what happens with duplicated rules? Are they combined into one rule?